contrib_bots/lib/joker.py

```diff
@@ -0,0 +1,52 @@
+# pip install schedule is necessary
+import schedule
+import time
+import json
+import threading
+import os
+from random import choice
+
+cache = os.path.expanduser('lib/Joker/jokes.json')
+
+def job(client):
```

Comment:
I think this function should be a part of the class.
Please move it inside and make it method.